orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/model/ManifestCoordinates.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.netflix.spinnaker.kork.annotations.NonnullByDefault;
+import lombok.Builder;
+import lombok.Value;
+
+@JsonDeserialize(builder = ManifestCoordinates.ManifestCoordinatesBuilder.class)
+@NonnullByDefault
+@Value
+public class ManifestCoordinates {
+  String kind;
+  String namespace;
+  String name;
```

Comment:
This is really just a personal preference, but I like to write out the `private final` even though `@Value` already does that for me (even though by default IntelliJ complains if you do, an inspection I've turned off).  I think it's probably because I'm not super comfortable with an annotation processor changing attributes on things I've written.  (It's already a little strange that Lombok injects stuff into your class, though I guess I've sort of gotten used to that...but actually changing the attributes of things I wrote is another level.)

I think also partly it's because this might get used as an example somewhere else and I'd like as many explicit `private final` things in our code to serve as good examples.